target/linux/ramips/image/Makefile

```diff
@@ -79,6 +79,17 @@ define Build/loader-kernel
 	$(call Build/loader-common,LOADER_DATA="$@")
 endef
 
+define Build/loader-okli-compile
+	$(call Build/loader-common,FLASH_OFFS=$(LOADER_FLASH_OFFS))
+endef
+
+# Arguments: <output name> <kernel offset>
```

Comment:
`<kernel offset>` is not really an offset, it is a `pad-to`!

@adrianschmutzler , is it better to correct the comment now at introduction to the `ramips` target, or it's better to change is treewide later?

Reply:
If you add something, it should be correct where you add it. You may still change it elsewhere, but that's no reason to add it incorrect first.

Reply:
Done. This was actually a copy from other code, but I 100% agree - best to have it correct at the start!